index.js

```diff
@@ -19,6 +19,7 @@ const Funnel = require('broccoli-funnel');
 const p = require('ember-cli-preprocess-registry/preprocessors');
 const fastbootTransform = require('fastboot-transform');
 const existsSync = fs.existsSync;
+const cheerio = require('cheerio');
```

Comment:
We should swap to using simple-html-tokenizer to parse, I'll try to whip up a demo to show what I mean.